tmt/cli.py

```diff
@@ -541,6 +541,9 @@ def tests_lint(
     '-t', '--template', metavar='TEMPLATE',
     help=f'Test template ({_test_templates}).',
     prompt=f'Template ({_test_templates})')
+@option(
+    '--link', metavar='LINK', multiple=True,
+    help='Link to the relevant issue.')
```

Comment:
I think the help now needs to be a little more descriptive, you are now accepting inputs in form of 'relation:target' and the order is pretty important here.
Maybe this could also be mentioned in the documentation? In docs/guide.rst is a Create section.

Reply:
Yes, metavar deserves an update, e.g. `[RELATION:]TARGET`

Reply:
Hi @KwisatzHaderach and @happz, the `help` is updated, please take a look, thanks!

Reply:
> Maybe this could also be mentioned in the documentation? In docs/guide.rst is a Create section.

@KwisatzHaderach and @happz , the `docs/guide.rst` is also updated, please help to review. If the description is not exactly what you want to have, please let me know, thanks!